__tests__/status.test.js

```diff
@@ -14,6 +17,7 @@ describe('Status', () => {
 
   beforeEach(async () => {
     await clearDatabaseState(app);
+    ({ cookies } = await getAuthenticatedUser(app));
```

Comment:
getAuthentificatedUser is used to retreive only cookie. Let's create some helper like "signIn" that would return auth cookie

Reply:
https://github.com/alex-popov-tech/backend-project-lvl4/issues/44